src/translate/compute.rs

```diff
@@ -26,16 +26,19 @@ pub fn translate_computation(meta: &TranslateMetadata, operation: ArithOp, left:
     match meta.arith_module {
         ArithType::BcSed => {
             let (left, right) = (left.unwrap_or_default(), right.unwrap_or_default());
-            let mut math_lib_flag = true;
-            // Removes trailing zeros from the expression
-            let sed_regex = "/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//";
+            let mut scale = "";
+            let mut after_bc = "";
+            let sed_regex = "| sed \"/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//\"";
             let op = match operation {
                 ArithOp::Add => "+",
                 ArithOp::Sub => "-",
                 ArithOp::Mul => "*",
-                ArithOp::Div => "/",
+                ArithOp::Div => {
+                    after_bc = sed_regex;
```

Comment:
You don't need a separate variable to store the "regex", you can assign a string literal here, like you do with `scale` below.  Also, can we please use a more descriptive term than `after_bc`, such as `strip_zeroes`?
```
ArithOp::Div => {
    strip_zeroes = "| sed \"/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//\"";
    "/"
},
```

Reply:
right now is strip zeros, maybe in the futurer is something else.
As there is still the issue with the escaping in the bash generated I am not sure if this PRs is fine as it is now.